tests/integration/targets/ios_snmp_server/tests/cli/merged.yaml

```diff
@@ -18,6 +18,14 @@
               traps:
                 - isis
               version: 2c
+          location: "in the racks"
```

Comment:
You might have to add/edit/update the list of commands it is asserting in the next tasks, that list lives in the vars folder, not sure how the integration tests are passing without that. Do check once again

Reply:
Done